src/commands/analyze.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+import {Analyzer} from 'polymer-analyzer';
+import {Element} from 'polymer-analyzer/lib/model/model';
+import {FSUrlLoader} from 'polymer-analyzer/lib/url-loader/fs-url-loader';
+import {PackageUrlResolver} from 'polymer-analyzer/lib/url-loader/package-url-resolver';
+import * as logging from 'plylog';
+import {Command} from './command';
+import {generateElementMetadata} from 'polymer-analyzer/lib/generate-elements';
+
+let logger = logging.getLogger('cli.command.analyze');
+
+export class AnalyzeCommand implements Command {
+  name = 'analyze';
+
+  description = 'Analyzes the input file with the Polymer Analyzer';
```

Comment:
I changed this to be slightly better, but since we only have a short description, it's limited right now. I think we should offer long descriptions and per-command help to fix this.